README.md

```diff
@@ -1,463 +1,476 @@
-  * _creation_ - `Date` - **modified** from construction to when the cookie was added to the jar
-  * _lastAccessed_ - `Date` - last time the cookie got accessed. Will affect cookie cleaning once implemented.  Using `cookiejar.getCookies(...)` will update this attribute.
+- _hostOnly_ - boolean - is this a host-only cookie (that is, no Domain field was set, but was instead implied).
+- _pathIsDefault_ - boolean - if true, there was no Path field on the cookie and `defaultPath()` was used to derive one.
+- _creation_ - `Date` - **modified** from construction to when the cookie was added to the jar.
+- _lastAccessed_ - `Date` - last time the cookie got accessed. Affects cookie cleaning after it is implemented. Using `cookiejar.getCookies(...)` updates this attribute.
 
-### `Cookie([{properties}])`
+#### `new Cookie([properties])`
 
-Receives an options object that can contain any of the above Cookie properties, uses the default for unspecified properties.
+Receives an options object that can contain any of the above Cookie properties. Uses the default for unspecified properties.
 
-### `.toString()`
+#### `.toString()`
 
-encode to a Set-Cookie header value.  The Expires cookie field is set using `formatDate()`, but is omitted entirely if `.expires` is `Infinity`.
+Encodes to a Set-Cookie header value. The Expires cookie field is set using `formatDate()`, but is omitted entirely if `.expires` is `Infinity`.
 
-### `.cookieString()`
+#### `.cookieString()`
 
-encode to a Cookie header value (i.e. the `.key` and `.value` properties joined with '=').
+Encodes to a Cookie header value (specifically, the `.key` and `.value` properties joined with `"="`).
 
-### `.setExpires(String)`
+#### `.setExpires(string)`
 
-sets the expiry based on a date-string passed through `parseDate()`.  If parseDate returns `null` (i.e. can't parse this date string), `.expires` is set to `"Infinity"` (a string) is set.
+Sets the expiry based on a date-string passed through `parseDate()`. If parseDate returns `null` (that is, can't parse this date string), `.expires` is set to `"Infinity"` (a string).
 
-### `.setMaxAge(number)`
+#### `.setMaxAge(number)`
 
-sets the maxAge in seconds.  Coerces `-Infinity` to `"-Infinity"` and `Infinity` to `"Infinity"` so it JSON serializes correctly.
+Sets the maxAge in seconds. Coerces `-Infinity` to `"-Infinity"` and `Infinity` to `"Infinity"` so it correctly serializes to JSON.
 
-### `.expiryTime([now=Date.now()])`
+#### `.expiryDate([now=Date.now()])`
 
-### `.expiryDate([now=Date.now()])`
-
-expiryTime() Computes the absolute unix-epoch milliseconds that this cookie expires. expiryDate() works similarly, except it returns a `Date` object.  Note that in both cases the `now` parameter should be milliseconds.
+`expiryTime()` computes the absolute unix-epoch milliseconds that this cookie expires. `expiryDate()` works similarly, except it returns a `Date` object. Note that in both cases the `now` parameter should be milliseconds.
```

Comment:
We seem to have dropped the `now` parameter for `expiryTime()`; can we bring that back? I think we can also document the call without the parameter.

Reply:
I think this is fine; removing this nit.